diff -Naur findchip.orig/findchip.c findchip/findchip.c
--- findchip.orig/findchip.c	2000-11-06 22:11:35.000000000 +0100
+++ findchip/findchip.c	2003-05-07 12:44:07.000000000 +0200
@@ -31,6 +31,7 @@
  ********************************************************************/
 
 #include <stdio.h>
+#include <stdlib.h>
 #include <unistd.h>
 #include <fcntl.h>
 #include <sys/io.h> 
diff -Naur findchip.orig/nsc.c findchip/nsc.c
--- findchip.orig/nsc.c	2000-11-06 22:11:36.000000000 +0100
+++ findchip/nsc.c	2003-05-07 12:45:11.000000000 +0200
@@ -30,6 +30,7 @@
 
 #include <sys/types.h>
 #include <stdio.h>
+#include <string.h>
 #include <unistd.h>
 #include <fcntl.h>
 #include <asm/io.h>
diff -Naur findchip.orig/smc.c findchip/smc.c
--- findchip.orig/smc.c	2000-11-06 22:11:37.000000000 +0100
+++ findchip/smc.c	2003-05-07 12:44:58.000000000 +0200
@@ -29,6 +29,7 @@
  ********************************************************************/
 
 #include <stdio.h>
+#include <string.h>
 #include <unistd.h>
 #include <fcntl.h>
 #include <asm/io.h>
